src/js.cookie.js

```diff
@@ -24,6 +24,13 @@
 		};
 	}
 }(function () {
+	var officialAttributes = {
```

Comment:
I understand that the intent here is to put the official attributes in the top do document what is official and what is not. In this case, though, we can try to reduce the number of bytes shifting this around since we already have tests and the public API docs that can document what is officially supported and what not.

It's ok to trade clean code for the sake of code size in this project.